src/style.css

```diff
@@ -1 +1,148 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  margin: 0;
+}
```

Comment:
It's recommended not to use tag names for styling (except `html` and `body`). Style elements using `.classes` and if needed with `:pseudo-classes`, `pseudo-elements` and `[attributes]`. For example, `.stars ul` could be replaced with a class like `.stars__list`.